src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -346,7 +346,7 @@ protected virtual void ValidateIdToken(OpenIdConnectProtocolValidationContext va
                 }
                 catch (Exception ex)
                 {
-                    throw LogHelper.LogExceptionMessage(new OpenIdConnectProtocolException(LogHelper.FormatInvariant(LogMessages.IDX21313, validationContext.ValidatedIdToken), ex));
+                    throw LogHelper.LogExceptionMessage(new OpenIdConnectProtocolException(LogHelper.FormatInvariant(LogMessages.IDX21313, validationContext.ValidatedIdToken.ToString()), ex));
```

Comment:
Again, it's better to wait until the last minute to do string operations for performance reasons. If our logging helper methods aren't able to handle this we should update the helper/utility methods not each instance where we call.